fundamentals/middleware/problem-details-service/problem-details-service.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
nit: I checked the other samples and to make it consistent maybe change the `csproj` file name to `ProblemDetailService.csproj` or something similar.

Reply:
Alright